microsoft-edge/webview2/concepts/browserfeatures.md

```diff
@@ -0,0 +1,83 @@
+*   The ability for features to be turned on \(or turned off\).  
+
+## Guiding Principles  
+
+Browser features within the context of WebView2 should adheres to the following guidelines for most WebView2 features.  
+
+*   Most of the WebView2 features work the same as Microsoft Edge features.  The following exceptions define why a feature is turned off.  
+    *   It doesn't make sense within the context of WebView2.  
+    *   It has an extended service cost.  
+*   The WebView2 features shouldn't include any Microsoft Edge branding.  
+    
+## Features  
+
+The following list displays the WebView2 features that differ from the new Microsoft Edge browser.  
+
+| Feature | Default | Configurable | Details |  
+|:--- |:--- |:--- | :--- | 
+| Autofill for Addresses | Enabled | Yes | This feature is turned on by default, you may turn it on or off. |  
+| Autofill for Passwords | Enabled | Yes | This feature is turned on by default, you may turn it on or off. |  
+| Autofill for Payments | Disabled | No | This feature is turned off. | 
+| Browser Extensions | Disabled | No | This feature is turned off. |   
+| Browser Task Manager | Disabled | No | This feature is turned off. |  
+| Continue-where-I-left-off prompt | Disabled | No | This feature is turned off. |  
+| Downloads | Enabled | Yes | This feature allows you to customize the download UI to manipulate downloads, such as block, redirect, save, pause, and so on.  For more information, navigate to <!--[download api][Webview2ReferenceDownloadApi]-->. |  
+| Family Safety | Disabled | No | This feature is turned off. |  
+| Hotkeys | Disabled | No | The following shortcuts aren't turned off:  `Ctrl`+`F` and `Ctrl`+`S` |  
+| IE Mode | Disabled | No | IE Mode is disabled. |  
+| Immersive Reader | Disabled | No | This feature is dependent on the browser UI for control and triggering.  This feature is turned off. |  
+| Intrusive Ads | Disabled | No | The UI for this feature is removed.  This feature is turned on. |  
+| Read Aloud | Disabled | No | The UI for this feature is removed.  This feature is turned on. |  
+| Smart Screen | Enabled | No | The UI for this feature is removed.  This feature is turned on. |  
+| Translate | Disabled | No | This feature is turned off. |  
+| Tracking Prevention | Disabled | No | The UI for this feature is removed.  This feature is turned on. |  
+| User Data Sync | Disabled | No | The feature that syncs your favorites, cookies, etc. between machines is disabled. |  
+| Windows Defender Application Guard | Disabled | No | The feature that syncs your favorites, cookies, etc. between machines is disabled. |  
+| edge:// URLs | See Details | Yes | Settings for the browser are displayed on webpages that use internal URLs.  Most of the webpages have Microsoft Edge branding or don't make sense within the context of WebView2.  For more details about the internal URLs that are turned off, navigate to [Blocked internal URLs](#blocked-internal-urls). |  
```

Comment:
edge:// URLs are not configurable by the app. The app can't turn on blocked URLs.